i2c/src/vhu_i2c.rs

```diff
@@ -114,7 +114,7 @@ impl<D: I2cDevice> VhostUserI2cBackend<D> {
     fn process_requests(
         &self,
         requests: Vec<I2cDescriptorChain>,
-        vring: Option<&VringRwLock>,
+        vring: &VringRwLock,
```

Comment:
The Option() on the VringRwLock is not mandatory and this patch is essentially a code cleanup, something the changelog should clearly state. I kept looking for a higher purpose to this work but in the end there isn't.

Reply:
Hmm, it was actually essential until this patch came in as I didn't figure out earlier that we can actually send a empty vring from tests to make it run and so the tests were rather passing None earlier.. And this is what the changelog says now:

    For tests we can pass a non related vring to just make it run, hence no
    need of making it optional. Always pass the vring.

I hoped it was alright.

Reply:
I have read the changelog several times again today and I still find it difficult to relate it to the code presented by the patch.  But that is my own interpretation, i.e what is clear for you might not be for me and vise versa.  In the end it is just terminology and that is why I didn't think is was necessary to delay approval of the patch.